client/network/src/protocol/generic_proto/handler/notif_out.rs

```diff
@@ -300,11 +304,12 @@ impl ProtocolsHandler for NotifsOutHandler {
 
 			NotifsOutHandlerIn::Send(msg) =>
 				if let State::Open { substream, .. } = &mut self.state {
-					if let Some(Ok(_)) = substream.send(msg).now_or_never() {
-					} else {
+					if !matches!(substream.send(msg).now_or_never(), Some(Ok(_))) {
 						log::warn!(
 							target: "sub-libp2p",
-							"📞 Failed to push message to queue, dropped it"
+							"📞 Queue of notifications with {} is full, dropped message (protocol: {:?})",
```

Comment:
```suggestion
							"📞 Notification queue with peer {} is full, dropped message (protocol: {:?})",
```

This sounds better to me. Not a native speaker though.